modules/ROOT/pages/ocis_release_notes.adoc

```diff
@@ -15,15 +15,71 @@ toc::[]
 
-Version 2.0.0 is a milestone in the development of Infinite Scale as it now comes with all features that have been planned for the upcoming general availability version. Before general availability, there are going to be a number of beta releases as the last development phase before Infinite Scale will be available and recommended for production use. 
+Infinite Scale 2.0.0 is the first stable release, ready for production scenarios.
 
-You can read more about the Infinite Scale beta phase on https://owncloud.com
+=== What is ownCloud?
+ownCloud is an open source server application that allows you to access your files from anywhere in a secure way. 
+The files are stored on a server running ownCloud. You can access your files via the browser or sync them to your desktop or mobile device like you might know it from oneDrive, Dropbox or others. The difference with ownCloud is, that you stay in controll of your data as you can install ownCloud on your own server.
 
-- https://owncloud.com/news/infinite-scale-beta-available/[Welcome to the Infinite Scale Beta]
-- https://owncloud.com/infinite-scale-beta[Infinite Scale Beta Page]
-- https://owncloud.com/news/infinite-scale-beta/[Infinite Scale Beta is coming]
+=== What is ownCloud Infinite Scale?
+ownCloud Infinite Scale is the latest generation of the ownCloud server application.
+With its modern architecture, Infinite Scale provides all deployment models for cloud infrastructure deployments and optimized scaling setups for the best output for the invested energy.
 
-**Notes on native client support**
+Please find the full list of changes here: https://github.com/owncloud/ocis/releases/tag/v2.0.0
+
+Amongst many other features, the most notable benefits of ownCloud Infinite Scale are:
+
+==== Spaces
+Spaces are special folders made for teamwork. 
+
+Members of a Space enjoy the service of the Space Manager, who takes care of enough quota and the right members of a Space. Thus a Space is the ideal place where school classes, business projects or working groups take place. Spaces don't "belong" to individual people, but to the role of the Space Manager. By that, ex. school classes can easily be handed over to the next teacher or organisations can transfer project resposibilities if an employee leaves the organisation.
+
+Usually Space Managers are for example project leaders or teachers. 
+All in all, Spaces make life easier for admins by giving Space Managers the ability to self-service.
+
+==== ownCloud Web
+The new design of ownCloud Web comes with a brand new look. But our actual goal is to make owncloud accessible to all people from children to experts. Technically ownCloud Web is build with VueJS as dedicated client, communicating via well-formed APIs with the Infinite Scale server
+Customizsation: ownCloud Web can get customized according to your corporate identity via config values.
+Please find the full list of changes here: https://github.com/owncloud/web/releases/tag/v6.0.0
```

Comment:
The beta phase had web releases 5.5.0, 5.6.0, 5.6.1, 5.7.0 and 6.0.0. What now, add all release links? Or only the most recent one?